src/Service/Config/ConfigGenerator.php

```diff
@@ -238,16 +244,12 @@ private function generateConfig(string $salesChannelId, string $carrier = ''): a
             }
         }
 
-        if (
-            $this->getConfigValue($salesChannelId, 'dropOffDays', $carrier) !== null && $this->isSettingEnabled($salesChannelId, 'dropOffDays', $carrier)
-        ) {
-            $settings["dropOffDays"] = implode(";", $this->getConfigValue($salesChannelId, 'dropOffDays', $carrier));
-        }
-
         if (!empty($this->getConfigString($salesChannelId, 'cutoffTime', $carrier)) && $this->isSettingEnabled($salesChannelId, 'cutoffTime', $carrier)
         ) {
-            $settings["cutoffTime"] = substr($this->getConfigString($salesChannelId, 'cutoffTime', $carrier), 0, -3);
+            $settings['cutoffTime'] = substr($this->getConfigString($salesChannelId, 'cutoffTime', $carrier), 0, -3);
         }
+        $settings['allowDeliveryOptions'] = true;
```

Comment:
als dit toch altijd true is, zet hem dan bovenaan in de initialisatie van $settings